### PR TITLE
Savestate: Remove unused exeception/variable

### PR DESCRIPTION
**Pull request by MrCK1**
Removes an exception that was never fully implemented to be used with a CRC check.

Removed an unused variable "m_didBios"

### CODE REVIEW

**Comment by LaneSh4d0w** [comment]
What's the utility of removing it if it can be implemented in the future ? For now it is not the main priority to do a CRC check, their priority resides in RPCS2's game motor and such.

**Comment by MrCK1** [comment]
> What's the utility of removing it if it can be implemented in the future ? For now it is not the main priority to do a CRC check, their priority resides in RPCS2's game motor and such.

The savestate is already checked against the game CRC (though only by filename). The only time I could see someone running into issues is if they intentionally renamed their savestate files to point to another game.

Also keep in mind that implementing the feature would also require changes to the CDVD plugin (which will hopefully be merged into the emulator at some point in the future)

**Comment by LaneSh4d0w** [comment]
I see. Indeed, if this is already implemented elsewhere in the code it is legitimate.

File: pcsx2/SaveState.h [hunk 1]
```diff
@@ -35,7 +35,7 @@ namespace Exception
 {
 	// ---------------------------------------------------------------------------------------
 	// Savestate Exceptions:
-	//   UnsupportedStateVersion / StateCrcMismatch
+	//   UnsupportedStateVersion
```
Comment (turtleli) [inline pcsx2/SaveState.h:38]: `UnsupportedStateVersion` also seems to be unused. Maybe remove it as well?

Comment (MrCK1) [reply pcsx2/SaveState.h]: Looks like you're right, changes pushed.

**Pull request merged**